src/monattack.cpp

```diff
@@ -2785,14 +2786,32 @@ bool mattack::photograph( monster *z )
         }
     }
 
-    if( z->friendly ) {
-        // Friendly (hacked?) bot ignore the player.
+    if( z->friendly || g->u.weapon.typeId() == "e_handcuffs" ) {
+        // Friendly (hacked?) bot ignore the player. Arrested suspect ignored too.
         // TODO: might need to be revisited when it can target npcs.
         return false;
     }
     z->moves -= 150;
     add_msg( m_warning, _( "The %s takes your picture!" ), z->name().c_str() );
     // TODO: Make the player known to the faction
+    if( one_in( 6 ) ) {
+        sounds::sound( z->pos(), 15, sounds::sound_t::alert,
+                       _( "a robotic voice boom, \"Citizen " ) + Name::generate( g->u.male ) + "!\"" );
```

Comment:
`string_format` would be better for I18N:
```cpp
string_format( _( "a robotic voice boom, \"Citizen  %s!\"" ), Name::generate( g->u.male ) )
```

Reply:
That's for strings with variables only. No need to do it for plain text strings.

Reply:
Please take a closer look at my example. `string_format` have several arguments: format string and the values for the placeholders.

Reply:
should have paid more attention to it from the beginning, thanks